### PR TITLE
Adding course links from www.learnhowtoprogram.com/courses

### PR DESCRIPTION
**Pull request by jspreddy**
Adding course links from here.

Epicodus,Inc.
www.learnhowtoprogram.com/courses

### CODE REVIEW

File: free-courses-en.md [hunk 1]
```diff
@@ -156,6 +160,7 @@
 * [Build Dynamic Websites](http://cs75.tv/2010/fall/)
 * [Code a personal splash page in 1 hour: A Hands-on Intro to HTML & CSS](http://www.thefirehoseproject.com/lessons/splash-page)
 * [Learn HTML5 Programming From Scratch](https://www.udemy.com/learn-html5-programming-from-scratch/)
+* [Epicodus Inc.: Learn how to program: CSS](https://www.learnhowtoprogram.com/css)
```
Comment (polyzen) [inline free-courses-en.md]: CSS is not C++. :p

**Comment by eshellman** [comment]
travis is complaining because it wants the resources in alphabetical order

**Comment by jspreddy** [comment]
Ready to merge.

**Comment by vhf** [comment]
Thanks!

**Pull request merged**